lib/ansible/modules/cloud/amazon/elb_classic_lb.py

```diff
@@ -1063,7 +1063,10 @@ def select_stickiness_policy(self):
                     if 'expiration' not in self.stickiness:
                         self.module.fail_json(msg='expiration must be set when type is loadbalancer')
 
-                    expiration = self.stickiness['expiration'] if self.stickiness['expiration'] is not 0 else None
+                    try:
+                        expiration = self.stickiness['expiration'] if int(self.stickiness['expiration']) else None
```

Comment:
Do you need expiration to be an int type as well or is it okay if it is a string repesentation (like:  "10" )